### PR TITLE
Keystore - new command

### PR DESCRIPTION
**Pull request by josefwijaya**
This command of the keystore help to generate new key for user

### CODE REVIEW

File: keystore/keystore.go [hunk 1]
```diff
@@ -1 +1,83 @@
...
+		if v == true {
+			kd := &KeyData{fname, lbl, addr, nil}
+			ks.Keys = append(ks.Keys, kd)
+		}
+	}
+
+	return ks
+}
+
+func (ks *Keystore) New(label, auth string, valAddr bool) (*KeyData, error) {
```
Comment (b00f) [inline keystore/keystore.go]: To make compatible with [key.EncryptKey(...) ](https://github.com/gallactic/gallactic/blob/b4ac9d41e87b2fe87ec7b06d9f23bb5a4e185713/keystore/key/encrypted.go#L160) function better to change the signature to `New (auth, lable string, ...)`
Please look here:

Comment (josefwijaya) [reply keystore/keystore.go]: I see what you mean. I'll make the changes in another commit

Comment (b00f) [reply keystore/keystore.go]: Thanks @josefwijaya

File: Makefile [hunk 2]
```diff
@@ -9,7 +9,7 @@ LDFLAGS= -ldflags "-X github.com/gallactic/gallactic/version.GitCommit=`git rev-
 CFLAGS=CGO_LDFLAGS="$(SPUTNIKVM_PATH)/c/libsputnikvm.a -ldl -lm"
 
 
-all: tools deps build install test test_release
+all: tools deps build install gtxkey test test_release
```
Comment (b00f) [inline Makefile]: Please rename `gtxkey` to `gallactic-keys`

File: cmd/gtxkey/main.go [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+import (
+	"github.com/abiosoft/ishell"
+	"github.com/gallactic/gallactic/common"
+	"github.com/gallactic/gallactic/keystore"
+	"github.com/gallactic/gallactic/version"
+)
+
+var title = `
+          __          __
```
Comment (b00f) [inline cmd/gtxkey/main.go]:
```
              .__  .__                 __  .__                  __                        
   _________  |  | |  | _____    _____/  |_|__| ____           |  | __ ____ ___.__. ______
  / ___\__  \ |  | |  | \__  \ _/ ___\   __\  |/ ___\   ______ |  |/ // __ <   |  |/  ___/
 / /_/  > __ \|  |_|  |__/ __ \\  \___|  | |  \  \___  /_____/ |    <\  ___/\___  |\___ \ 
 \___  (____  /____/____(____  /\___  >__| |__|\___  >         |__|_ \\___  > ____/____  >
/_____/     \/               \/     \/             \/               \/    \/\/         \/ 
```

**Pull request merged**